src/main/flight/pid_mwrewrite.c

```diff
@@ -109,18 +111,36 @@ STATIC_UNIT_TESTED int16_t pidMultiWiiRewriteCore(int axis, const pidProfile_t *
         DTerm = 0;
     } else {
         // delta calculated from measurement
-        int32_t delta = -(gyroRate - lastRateForDelta[axis]);
-        lastRateForDelta[axis] = gyroRate;
-        // Divide delta by targetLooptime to get differential (ie dr/dt)
-        delta = (delta * ((uint16_t)0xFFFF / ((uint16_t)targetLooptime >> 4))) >> 6;
-        if (pidProfile->dterm_cut_hz) {
-            // DTerm delta low pass filter
-            delta = lrintf(applyBiQuadFilter((float)delta, &deltaFilterState[axis]));
+        int32_t delta;
+        if (pidProfile->dterm_noise_robust_differentiator) {
+            // Calculate derivative using 5-point noise-robust differentiator without time delay (one-sided or forward filters)
+            // by Pavel Holoborodko, see http://www.holoborodko.com/pavel/numerical-methods/numerical-derivative/smooth-low-noise-differentiators/
+            // h[0] = 5/8, h[-1] = 1/4, h[-2] = -1, h[-3] = -1/4, h[-4] = 3/8
+            delta = 5*gyroRate + 2*lastRate[axis][0] - 8*lastRate[axis][1] - 2*lastRate[axis][2] + 3*lastRate[axis][3];
```

Comment:
So rewrite is not configurable. It is hardcoded to 5point differentiator? Why the choice for that?